specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+        },
+        "objectsTargetedCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects that meet the condition as defined in the storage task assignment execution context. Filter options such as objectsTargetedCount gt 50 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectsOperatedOnCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects that meet the storage tasks condition and were operated upon. Filter options such as objectsOperatedOnCount ge 100 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectFailedCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects where task operation failed when was attempted. Filter options such as objectFailedCount eq 0 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectsSucceededCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects where task operation succeeded when was attempted.Filter options such as objectsSucceededCount gt 150 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "runStatusError": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Well known Azure Storage error code that represents the error encountered during execution of the run instance."
+        },
+        "runStatusEnum": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "InProgress",
+            "Finished"
+          ],
+          "x-ms-enum": {
+            "name": "RunStatusEnum",
+            "modelAsString": true
+          },
+          "description": "Represents the status of the execution."
+        },
+        "summaryReportPath": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Full path to the verbose report stored in the reporting container as specified in the assignment execution context for the storage account. "
+        },
+        "taskId": {
```

Comment:
>taskId [](http://example.com/codeflow?start=9&length=6)

Given that this id corresponds to an ARM resource, use this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

Reply:
done